docs/examples/ITest.ipynb

```diff
@@ -0,0 +1,199 @@
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 2,
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Example with the iTest from Bilt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "import qcodes as qc\n",
+    "from qcodes_contrib_drivers.drivers.Bilt.ITest import ITest"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Load instrument\n",
+    "\n",
+    "Here the import is done by USB."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": "Connected to: 2142 \"ITEST BE2142B/12V 15mA DC-SOURCE/SN04-016 LC1846 VL446\\240\" (serial:None, firmware:None) in 3.63s\n"
+    }
+   ],
+   "source": [
+    "dac = ITest(name='dac',\n",
+    "            address='ASRL4::INSTR',\n",
+    "            init_start=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initiallization"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "for channel in dac.channels:\n",
+    "    channel.v(0)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Measurement\n",
+    "\n",
+    "User can loop over the channel"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "output_type": "stream",
+     "name": "stdout",
+     "text": "channel 1 voltage 9.76e-08 V\nchannel 2 voltage 5.29e-07 V\nchannel 3 voltage 1.45e-08 V\nchannel 4 voltage 3.26e-07 V\nchannel 5 voltage 1.49e-07 V\nchannel 6 voltage -5.21e-07 V\nchannel 7 voltage -9.63e-08 V\nchannel 8 voltage 7.69e-07 V\nchannel 9 voltage 3.86e-07 V\nchannel 10 voltage 1.37e-06 V\nchannel 11 voltage 3.42e-07 V\nchannel 12 voltage -1.11e-06 V\nchannel 13 voltage 9.32e-07 V\nchannel 14 voltage 1.87e-07 V\nchannel 15 voltage 1.43e-07 V\nchannel 16 voltage -2.42e-07 V\n"
+    }
+   ],
+   "source": [
+    "for channel in dac.channels:\n",
+    "    print('channel {} voltage {:.2e} V'.format(channel.chan_num, channel.v()))"
+   ]
+  },
+  {
+   "source": [
+    "## Set\n",
+    "We can set a voltage on a specific channel:"
+   ],
+   "cell_type": "markdown",
+   "metadata": {}
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": "(None, 0.50000103)"
+     },
+     "metadata": {},
+     "execution_count": 5
+    }
+   ],
+   "source": [
+    "dac.ch04.v(0.5), dac.ch04.v()"
```

Comment:
Perhaps it's better to split each command in one cell, because in this way, it returns None for dac.ch04.v(0.5), which is misleading.